docker-compose.yml

```diff
@@ -1,24 +1,128 @@
+    # Cold boots will be faster and you'll be able to scale your DB independently of your app.
+    # @see https://www.prisma.io/docs/data-platform/data-proxy/use-data-proxy
+    # PRISMA_GENERATE_DATAPROXY: true
+    PRISMA_GENERATE_DATAPROXY:
+
+    ###############
+    #  NEXT AUTH  #
+    ###############
+
+    # @see: https://next-auth.js.org/configuration/options#nextauth_secret
+    # You can use: `openssl rand -base64 32` to generate one
+    NEXTAUTH_SECRET: RANDOM_STRING
+
+    # Set this to your public-facing URL, e.g., https://example.com
+    # You do not need the NEXTAUTH_URL environment variable in Vercel.
+    NEXTAUTH_URL: http://localhost:3000
+
+    # If you encounter NEXT_AUTH URL problems this should always be localhost:3000 (or whatever port your app is running on)
+    # NEXTAUTH_URL_INTERNAL: http://localhost:3000
+
+    ################
+    #  MAIL SETUP  #
+    ################
+
+    # Necessary if email verification and password reset are enabled.
+    # See optional configurations below if you want to disable these features.
+
+    MAIL_FROM: noreply@example.com
+    SMTP_HOST: localhost
+    SMTP_PORT: 1025
+    SMTP_SECURE_ENABLED: 0 # Enable for TLS (port 465)
+    SMTP_USER: smtpUser
+    SMTP_PASSWORD: smtpPassword
+
+
+    ########################################################################
+    # ------------------------------ OPTIONAL -----------------------------#
+    ########################################################################
+
+    # Uncomment the variables you would like to use and customize the values.
+
+    #####################
+    # Disable Features  #
+    #####################
+
+    # Email Verification. If you enable Email Verification you have to setup SMTP-Settings, too.
+    # NEXT_PUBLIC_EMAIL_VERIFICATION_DISABLED: 1
+
+    # Password Reset. If you enable Password Reset functionality you have to setup SMTP-Settings, too.
+    # NEXT_PUBLIC_PASSWORD_RESET_DISABLED: 1
+
+    # Signup. Disable the ability for new users to create an account.
+    # NEXT_PUBLIC_SIGNUP_DISABLED: 1
+
+    ##########
+    # Other  #
+    ##########
+
+    # Display privacy policy, imprint and terms of service links in the footer of signup & public pages.
+    NEXT_PUBLIC_PRIVACY_URL: ""
+    NEXT_PUBLIC_TERMS_URL: ""
+    NEXT_PUBLIC_IMPRINT_URL: ""
+
+    # Disable Sentry warning
+    SENTRY_IGNORE_API_RESOLUTION_ERROR: 1
+
+    # Enable Sentry Error Tracking
+    NEXT_PUBLIC_SENTRY_DSN: ""
+
+    # Configure Github Login
+    NEXT_PUBLIC_GITHUB_AUTH_ENABLED: 0
+    GITHUB_ID: ""
+    GITHUB_SECRET: ""
+
+    # Configure Google Login
+    NEXT_PUBLIC_GOOGLE_AUTH_ENABLED: 0
+    GOOGLE_CLIENT_ID: ""
+    GOOGLE_CLIENT_SECRET: ""
+
+
+    # Stripe Billing Variables
+    NEXT_PUBLIC_STRIPE_PRICING_TABLE_ID: ""
+    NEXT_PUBLIC_STRIPE_PUBLIC_KEY: ""
+    STRIPE_SECRET_KEY: ""
+    STRIPE_WEBHOOK_SECRET: ""
+
+
 services:
   postgres:
-    restart: unless-stopped
```

Comment:
If anything try `restart: always` the user would typically want to manage their restart policy themselves if they even know the feature exists.

Reply:
I think having this option in the docker compose is a good default for users who aren't that familiar with docker. If experienced users want to override that option, that's fine, but if for some reason the inexperienced users' server stops running, that's not good.